parser.y

```diff
@@ -957,6 +958,8 @@ import (
 	WhenClauseList		"When clause list"
 	WithReadLockOpt		"With Read Lock opt"
 	WithGrantOptionOpt	"With Grant Option opt"
+	WithValidation		"with validation"
+	WithValidationOpt		"optional with validation"
```

Comment:
The indention here is weird. Would you please fix it?

Reply:
fine, but is there any format tool for parser.y?

Reply:
unfortunatly, not